Lib/idlelib/outwin.py

```diff
@@ -1,43 +1,115 @@
+"""Editor window that can serve as an output file.
+"""
+
 import re
 
-from tkinter import *
-import tkinter.messagebox as tkMessageBox
+from tkinter import messagebox
 
 from idlelib.editor import EditorWindow
 from idlelib import iomenu
 
 
-class OutputWindow(EditorWindow):
+file_line_pats = [
```

Comment:
@csabella dh, I saw it. I think it should be another issue to deal with these codes changed.